nanome_chemical_properties/PropertiesHelper.py

```diff
@@ -30,6 +41,41 @@ def __init__(self):
             ('AR', '# Aromatic Rings', '%d', mDesc.CalcNumAromaticRings)
         ]
 
+        if not os.path.exists(API_SETTINGS):
+            return
+
+        with open(API_SETTINGS, 'r') as f:
+            self.api = json.load(f)
+
+        if self.api.get('overwrite'):
+            self._properties = []
```

Comment:
Could be cool in the future to replace properties with this flag instead of clearing them